make.jl

```diff
@@ -24,53 +73,35 @@ end
 
 # You can skip this part if you are on a metered
 # connection by calling `julia make.jl no-tutorials`
-tutorial_path = joinpath(@__DIR__, "_tutorials")
+tutorial_path = joinpath(tmp_path, "_tutorials")
 in("no-tutorials", ARGS) || copy_tutorial(tutorial_path)
 
 # set default baseurl for the master branch
-baseurl = "/dev"
-
-# deploy
-devurl = "dev"
-tags = tag_list(GitRepo(package_directory))
-versions = VersionNumber.(tags)
-
-# Find the most recent tag
-function highest_tags(versions)
-    majors = unique(map(x -> x.major, versions))
-    tags_to_use = []
-
-    for major in majors
-        filtered = filter(x -> x.major == major, versions)
-        minors = unique(map(x -> x.minor, filtered))
-
-        for minor in minors
-            minor_filtered = sort(filter(x -> x.minor == minor, filtered))
-            
-            push!(tags_to_use, minor_filtered[end])
-        end
-    end
-    return sort(tags_to_use)
-end
+baseurl = "/" * version
 
-highest = maximum(highest_tags(versions))
+@info "" baseurl
```

Comment:
Maybe @debug instead?

Reply:
Well, I'd rather be able to see it in the build logs without setting any other environment variables. It's pretty rare that this would be run by a human.